apps/happy-blocks/index.php

```diff
@@ -62,18 +73,67 @@ function a8c_happyblocks_assets() {
  * Load front-end view assets.
  */
 function a8c_happyblocks_view_assets() {
-	if ( ! is_admin() ) {
 		$assets = require plugin_dir_path( __FILE__ ) . 'dist/view.min.asset.php';
 
 		$style_file = 'dist/view' . ( is_rtl() ? '.rtl.css' : '.css' );
 		wp_enqueue_style(
-			'a8c-happyblocks-css',
+			'a8c-happyblock-view-css',
 			plugins_url( $style_file, __FILE__ ),
 			array(),
 			$assets['version']
 		);
-	}
-}
 
+		$script_file = 'dist/view.js';
+		wp_enqueue_script(
+			'a8c-happyblock-view-js',
+			plugins_url( $script_file, __FILE__ ),
+			array( 'wp-element' ),
+			$assets['version'],
+			true
+		);
+}
 add_action( 'enqueue_block_editor_assets', 'a8c_happyblocks_assets' );
 add_action( 'wp_enqueue_scripts', 'a8c_happyblocks_view_assets' );
+
+/**
+ * Get the pricing plans configuration
+ *
+ * @return array
+ */
+function a8c_happyblocks_get_config() {
+	$domain = wp_parse_url( get_site_url() )['host'];
```

Comment:
I don't have the full context here but wouldn't `home_url` return the same here?